$NetBSD$
(1)
The original has problem finding _fini symbol resulting
wrong format of binary executable.

(2)
Why the name has -2 suffix ?
patches/patch-ab also modify configure.in

--- configure.in.orig	2011-09-24 18:33:08.000000000 +0900
+++ configure.in	2011-09-24 19:36:36.000000000 +0900
@@ -1026,13 +1026,8 @@
     START_FILES='pre-crt0.o $(CRT_DIR)/crt0.o'
     ;;
   netbsd | openbsd )
-    if test -f $CRT_DIR/crti.o; then
 	LIB_STANDARD='-lgcc -lc -lgcc $(CRT_DIR)/crtend.o $(CRT_DIR)/crtn.o'
-        START_FILES='pre-crt0.o $(CRT_DIR)/crt0.o $(CRT_DIR)/crti.o $(CRT_DIR)/crtbegin.o'
-    else
-	LIB_STANDARD='-lgcc -lc -lgcc $(CRT_DIR)/crtend.o'
-	START_FILES='pre-crt0.o $(CRT_DIR)/crt0.o $(CRT_DIR)/crtbegin.o'
-    fi
+	START_FILES='pre-crt0.o $(CRT_DIR)/crt0.o $(CRT_DIR)/crti.o $(CRT_DIR)/crtbegin.o'
     ;;
 esac
 
